### PR TITLE
Clean up a few hacks

### PR DESCRIPTION
**Pull request by gaearon**
Nothing important.

### CODE REVIEW

File: packages/react-devtools-core/src/backend.js [hunk 1]
```diff
@@ -17,13 +17,6 @@ type ConnectOptions = {
   isAppActive?: () => boolean,
 };
 
-// TODO: why?
-if (!window.performance) {
```
Comment (gaearon) [inline packages/react-devtools-core/src/backend.js]: The only place depending on `performance.now` seems to be Relay plugin so inlined this.

File: packages/react-devtools-core/src/backend.js [hunk 2]
```diff
@@ -36,6 +29,7 @@ installGlobalHook(window);
 installRelayHook(window);
 
 if (window.document) {
+  // This shell is universal, and might be used inside a web app.
```
Comment (gaearon) [inline packages/react-devtools-core/src/backend.js]: This wasn't obvious.

File: shells/chrome/src/backend.js [hunk 3]
```diff
@@ -15,15 +15,8 @@ var BananaSlugBackendManager = require('../../../plugins/BananaSlug/BananaSlugBa
 var Bridge = require('../../../agent/Bridge');
 var inject = require('../../../agent/inject');
 var setupHighlighter = require('../../../frontend/Highlighter/setup');
-var setupRNStyle = require('../../../plugins/ReactNativeStyle/setupBackend');
 var setupRelay = require('../../../plugins/Relay/backend');
 
-// TODO: check to see if we're in RN before doing this?
-setInterval(function() {
```
Comment (gaearon) [inline shells/chrome/src/backend.js]: This is Chrome shell. We can't be in RN, and Chrome shell hasn't work on for a long time.

**Comment by jhen0409** [comment]
I think we can keep the `ReactNativeStyle` plugin in Chrome part, it can be used for [`react-native-web`](https://github.com/necolas/react-native-web) (need assign [`resolveRNStyle`](https://github.com/necolas/react-native-web/blob/master/src/apis/StyleSheet/flattenStyle.js) to hook), that will be useful because the style editor in `Elements` tab of devtools is very complicated for RNW.

**Comment by gaearon** [comment]
Did it work before? Could you open a new issue about it?

**Comment by jhen0409** [comment]
OK, I opened the issue #634.

**Pull request merged**